Python/ExcelOutputClass.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+Spyder Editor
+
+This is a temporary script file.
+"""
+
+import os
+import xlsxwriter
+import openpyxl
+
+os.chdir(r'\Users\Ben\Documents\GitHub\Ben-Projects')
+
+def make_excel_file(title):
+        workbook = xlsxwriter.Workbook(title)
+        worksheet = workbook.add_worksheet()
+        worksheet.write(0,0, "JOURNAL")
+        worksheet.write(0,1, "DATE")
+        worksheet.write(0,2, "ACCOUNT")
+        worksheet.write(0,3, "CONTROL #")
+        worksheet.write(0,4, "DOCUMENT #")
+        worksheet.write(0,5, "REFERENCE #")
+        worksheet.write(0,6, "AMOUNT")
+        worksheet.write(0,7, "DESCRIPTION")
+        return workbook.filename
+        
+def add_row(wb1,row, column, date,account,controlnumber,documentnumber,amount,description):
```

Comment:
You should be able to make the same general assumption (function contract) that requires the "Caller" is required to order their data according to the header data in the make excel function.